.github/workflows/ppc64le-image.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         uses: redhat-actions/buildah-build@v2
         with:
           image: operator-pipelines-images
-          tags: ppc64le ${{ github.sha }}
+          tags: ppc64le-${{ github.sha }}
```

Comment:
This change has the side effect of removing a floating tag from ppc64le images. Could you keep ppc64le tag and add a separate one `ppc64le-sha`?

Reply:
Floating tag is pointless, as long as the ppc64le image is built on every commit. WDYT?

Reply:
@mayurwaghmode what should we do with this tag?

Reply:
@MarcinGinszt Can we have a tag as ppc64le:latest?

Reply:
Yes, we can. But would you like to build it for every commit on every branch, or maybe only for master?

Reply:
what if we set the ppc64le-$BRANCH-${{ github.sha }} for every branch and ppc64le:$RELEASE for every new release?

Reply:
@rpsene @mayurwaghmode  You are encouraged to submit the PR in order to create a custom image tagging system. I just want to unblock us by removing the bug, that overrides images necessary for prod.

Reply:
@rpsene @mayurwaghmode  can we merge this one?

Reply:
@MarcinGinszt for now yes. We will update it later.